pkg/controllers/statusaggregator/controller.go

```diff
@@ -353,7 +362,7 @@ func (a *StatusAggregator) clusterObjs(qualifiedName common.QualifiedName) (map[
 		)
 		if err != nil {
 			wrappedErr := errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, cluster.Name)
-			utilruntime.HandleError(wrappedErr)
+			logger.Error(wrappedErr, "")
```

Comment:
How about adding an error log here？
```suggestion
			logger.WithValues("target-kind", targetKind, "object", key, "cluster-name", cluster.Name).Error(err, "Failed to get object from cluster")
```

Reply:
Please add other error messages if missing.

Reply:
get